pymc3/plots/compareplot.py

```diff
@@ -58,7 +58,7 @@ def compareplot(comp_df, insample_dev=True, se=True, dse=True, ax=None,
         yticks_labels[0] = comp_df.index[0]
         yticks_labels[2::2] = comp_df.index[1:]
         ax.set_yticks(yticks_pos)
-        ax.errorbar(x=comp_df.loc[1:, ic],
+        ax.errorbar(x=comp_df[ic][1:],
```

Comment:
should this be `.iloc` instead?

Reply:
This will give the correct result (all elements except the first one), but I could change it to `.iloc` to make it more explicit.

Reply:
For this line `.iloc` is optional but not for the other one! I just changed both lines. Thanks

Reply:
Oh, you might be right. I actually meant `comp_df.iloc[1:, ic]` but `iloc` expects both indices (index and column) to be an integer, so this is probably the only way.